lambda/latest_image_tracker/template.yml

```diff
@@ -24,14 +24,17 @@ Resources:
           Properties:
             Topic: !Ref ImageBuilderSNSTopic
       Policies:
-          - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+          # - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
           - Version: '2012-10-17'
             Statement:
               - Effect: Allow
                 Action:
                   - ssm:PutParameter
                   - ssm:AddTagsToResource
                   - ssm:GetParameters
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
```

Comment:
As SAM is adding the `AWSLambdaBasicExecutionRole`, these policies can also be removed.